site/docsource/Installation.adoc

```diff
@@ -16,7 +16,7 @@ is installed, run the following command:
 
 [source,sh]
 ------------------------------
-npm install --save bs-platform
+npm install --save-dev bs-platform
```

Comment:
The docs ought to explain the merits of each option. Modules written for use with other bucklescript modules ought to include `bs-platform` as a dependency, otherwise it ought to be included as a dev dependency. It's about gauging what fraction of users will benefit more from a light install process versus the advantages of compilation on many interacting modules.